common/test/acceptance/pages/studio/settings.py

```diff
@@ -70,6 +70,37 @@ def get_element(self, css_selector):
         results = self.get_elements(css_selector=css_selector)
         return results[0] if results else None
 
+    def set_value(self, dates_dictionary):
```

Comment:
nit: this method could be used for items other than dates, so it would be better to name the method something like "set_element_values" and the argument something like "element_values"

The docstring could read something like:
Set the values of the elements to those specified in the element_values dict.